packages/dcos-integration-test/extra/test_tweeter_demo.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import os
+import time
+
+import pytest
+import requests
+
+import test_helpers
+
+from test_packaging import (
+    skipif_insufficient_resources
+)
+
+log = logging.getLogger(__name__)
+
+DEMO_RESOURCE_REQUIREMENTS = {
+    'number_of_nodes': 5,
+    'node': {
+        'disk': 0,
+        'mem': 0,
+        'cpus': 1
+    }
+}
+
+
+def _get_tweeter_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/tweeter.json')
+    return response.json()
+
+
+def _get_post_tweets_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/post-tweets.json')
+    return response.json()
+
+
+def _get_tweet_count(address):
+    """Count the number of tweets on the tweeter app's webpage
+
+    Args:
+        address: str URL including http://
```

Comment:
Just `address: str URL` is fine. No need for adding 'http://`, because It can be `https:` too.
Also, what does `response.text` contain? How does counting `tweet-content` return the count?

Reply:
The app is defined so that each tweet's text is stored in a `tweet-content` html tag. Response.text is the text of the app's page source.